src/bits/utils/create_new_instrument.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
```

Comment:
Why can't we have a template directory for the instrument?  We used to have such an installer (in bluesky_training repo) and moved away from that idea.  Now, this moves us back to it.